softserve/views.py

```diff
@@ -149,3 +150,11 @@ def delete(vid=None):
         delete_node.delay(name)
         flash('Deleting {} machine'.format(name))
     return redirect('/dashboard')
+
+
+@app.route('/report')
+def report():
+    vmList = Vm.query.join(NodeRequest).join(User) \
```

Comment:
This query looks a bit weird. Do you know what the performance implications of this would be?